Frontend/Frontend.Client/Pages/LoginPage.razor

```diff
@@ -33,32 +57,78 @@
 @code
 {
     private ClientWebSocket? _webSocket;
+    string? errorMesage;
     public LoginInfo userInfo { get; set; } = new LoginInfo();
+    // the purpose of these booleans is to be used in an if statement in the HTML to render the error codes accordingly
```

Comment:
generally if the code needs a comment to explain it then its not clear enough or its a very weird edge case, check if this comment is needed to understand the code or not otherwise you can remove it.